common/web/types/src/consts/modifier-key-constants.ts

```diff
@@ -89,3 +118,64 @@ export function translateVisualKeyboardShiftToLdmlModifiers(shift: VisualKeyboar
 
   return mod;
 }
+
+
+
+function VkShiftStateToKmxShiftState(ShiftState: number): number {
+
+  interface TVKToKMX {
+    VK: VisualKeyboardShiftState; KMX: ModifierKeyConstant;
+  }
+
+  const Map: TVKToKMX[] = [
```

Comment:
`Map` might not be the best name here - it's not really a map, and it conflicts with the `Map` type.

Reply:
This is a move-only refactor so I haven't changed any of the code within. Will leave this for a future cleanup.